plugins/event_source/kafka.py

```diff
@@ -28,20 +36,37 @@ async def main(queue: asyncio.Queue, args: Dict[str, Any]):
     topic = args.get("topic")
     host = args.get("host")
     port = args.get("port")
+    cafile = args.get("cafile")
+    certfile = args.get("certfile")
+    keyfile = args.get("keyfile")
+    password = args.get("password")
+    check_hostname = args.get("check_hostname", True)
     group_id = args.get("group_id", None)
     offset = args.get("offset", "latest")
 
     if offset not in ("latest", "earliest"):
         raise Exception(f"Invalid offset option: {offset}")
 
+    if cafile:
+        context = create_ssl_context(
+            cafile=cafile,
+            certfile=certfile,
+            keyfile=keyfile,
+            password=password,
+        )
+        context.check_hostname=check_hostname
```

Comment:
Apologies @morvencao, we need whitespace around the operator to fix the linting issue.

Reply:
done